src/Build/Instance/TaskRegistry.cs

```diff
@@ -1765,6 +1739,7 @@ public void Translate(ITranslator translator)
                 translator.Translate(ref _taskFactoryAssemblyLoadInfo, AssemblyLoadInfo.FactoryForTranslation);
                 translator.Translate(ref _taskFactory);
                 translator.Translate(ref _parameterGroupAndTaskBody);
+                translator.Translate(ref _registrationOrderId);
```

Comment:
How will the registration ID compare to tasks registered in another process after this instance is deserialized there? Are serialized and then deserialized task registries effectively read-only?

Reply:
Oh - good question - let's make sure we do not have to have answer for it :-) (I'll move the counter to TaskRegistry and update it on deserialization)